### PR TITLE
Add EditorConfig file

### PR DESCRIPTION
**Pull request by AlexWayfer**
More info [here](http://editorconfig.org/)

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = tab
```
Comment (AlexWayfer) [inline .editorconfig]: There is also can be `indent_size = N` (for alignment, [article about this](https://dmitryfrank.com/articles/indent_with_tabs_align_with_spaces)), but I don't know the preferred size (and I didn't see alignment of code in this project at all).

**Comment by Jguer** [comment]
I write using spaces and indent 4. But even then, this is half useless because all of our code is supposed to be formatted with gofmt, what we should force is gofmt before committing

**Comment by AlexWayfer** [comment]
OK… I have some troubles with my environment: Atom Editor + EditorConfig plugin (which is responsible for indentation, newlines and trailing spaces) + disabled Whitespace plugin (core default plugin for these things). I switched from Workspace to EditorConfig because every project has its own style and convention about previously mentioned things. But `gofmt` is something new for me (I never seen such automatic thing before).

I'll think about it. You can close this PR if you want.

> I write using spaces and indent 4.

But code has tabs… and we can not specify indentation size, or even style. And config will have information about trailing spaces, newlines, etc.

**Comment by Jguer** [comment]
https://atom.io/packages/go-plus seems nice to integrate gofmt into the workflow.

**Comment by AlexWayfer** [comment]
> https://atom.io/packages/go-plus seems nice to integrate gofmt into the workflow.

Yeah, seems working, thank you.

**Pull request closed without merging**